### PR TITLE
PSU card implementation in MFT

### PR DESCRIPTION
**Pull request by frmanso**

### CODE REVIEW

File: Detectors/ITSMFT/MFT/base/include/MFTBase/PowerSupplyUnit.h [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#include "TNamed.h"
+#include "TGeoVolume.h"
+#include "TGeoMatrix.h"
+
+namespace o2
+{
+namespace mft
+{
+
+class PowerSupplyUnit : public TNamed
```
Comment (sawenzel) [inline Detectors/ITSMFT/MFT/base/include/MFTBase/PowerSupplyUnit.h]: Is there a strong need to derive from TNamed? I believe we can keep things simple and live without inheritance.

**Comment by bovulpes** [comment]
In some classes TNamed is not used, indeed and can be taken out. This is the list of those classes:

GeometryBuilder, HalfCone, Flex, Support, HeatExchanger, Segmentation, VSegmentation and the 
last added one PowerSupplyUnit.

In other classes the name is used to give a name to the associated assembly:

HalfDetector, HalfDisk, Ladder

and the main class Geometry is using TNamed in order to get a name and a title.

**Comment by sawenzel** [comment]
Fine. I guess this it is not critical. Just wanted to raise awareness that this inheritance might not be necessary. I suggest we go ahead and you take out all TNames in a dedicated PR?

**Comment by bovulpes** [comment]
Ok, Franck will do it in a dedicated PR.

**Pull request merged**